atom/browser/browser.h

```diff
@@ -158,6 +174,12 @@ class Browser : public WindowListObserver {
   // Add a custom task to jump list.
   bool SetUserTasks(const std::vector<UserTask>& tasks);
 
+  // Get the current Jump List settings.
+  v8::Local<v8::Value> GetJumpListSettings(mate::Arguments* args);
```

Comment:
Ah OK, I wasn't clear on the distinction between Browser and App, thanks for explaining it.